atm.js

```diff
@@ -6,16 +6,30 @@ var checkingDeposit = function(amount){
 
 }
 
-function checkingWithdrawal(amount){
+function bank(originalAmount){
+  originalAmount = originalAmount 0;
+  this.checking = originalAmount.checking;
+  this.savings = originalAmount.savings;
+  this.checking = parseInt(checking)
+  this.savings = parseInt(savings)
```

Comment:
I see the start of some expectations here, but don't see the originalAmount object.  That object could ensure these values are integers.